torch/utils/_cuda_trace.py

```diff
@@ -0,0 +1,77 @@
+from typing import Any, Callable, Generic, List, TypeVar
+import logging
+from typing_extensions import ParamSpec
```

Comment:
Usually the import order is standard library imports, then third-party imports, then first-party imports, all in separate blocks, separated by blank lines. Hence in this case the `typing_extensions` one would go separate.

Also, within blocks, usually the `import foo` lines go all together above all the `from foo import bar` lines. And within these two groups imports are sorted alphabetically. 

These are also the conventions of isort (https://pycqa.github.io/isort/), which is kinda used at Meta, though I don't know how much PyTorch follows this.

Reply:
I'm sorry, I forgot `typing_extensions` is not standard.